packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -73,11 +73,12 @@ function Tab({
       aria-expanded={selected}
       {...props}
     >
-      {selected && (
-        <AriaAnnouncer id="announcer" aria-live="assertive">
-          {`${simpleName} ${announcerText}`}
-        </AriaAnnouncer>
-      )}
+      {selected &&
+        (simpleName.length > 0 || announcerText.length > 0) && (
+          <AriaAnnouncer id="announcer" aria-live="assertive">
+            {`${simpleName} ${announcerText}`}
+          </AriaAnnouncer>
+        )}
```

Comment:
Is the styling of this because of prettier? I think we'd be better served having all the logic on the first line.

```js
{selected && (simpleName.length || announcerText.length) && (
  <AriaAnnouncer id="announcer" aria-live="assertive">
    {`${simpleName} ${announcerText}`}
  </AriaAnnouncer>
)}
```

Reply:
Yes, prettier is responsible for that. :\

Reply:
I would say it's safe to let prettier do it's thing. It doesn't make a difference and is still easy enough to read imo.

Reply:
I don't disagree. Some of the things prettier does are interesting, this instance being the least of them.